anaconda.py

```diff
@@ -81,6 +81,13 @@ def exitHandler(rebootData):
         task_proxy = STORAGE.get_proxy(task_path)
         sync_run_task(task_proxy)
 
+    # reenable LVM auto-activation disabled in enable_installer_mode
+    from blivet.devicelibs import lvm
+    try:
+        lvm.reenable_lvm_autoactivation()
+    except Exception as e:  # pylint: disable=broad-except
+        log.error("Failed to disable LVM auto-activation: %s", str(e))
```

Comment:
Just a tiny fix, 'Failed to reenable..' 

Please rebase and I can merge this.

Reply:
Also you can except RuntimeError, I see that's what you throw in blivet. 
So that you dont have to use the broad-except.